jenkins-schema.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./jenkins.sh
```

Comment:
Ah, scratch that, the significant feature is calling gh-status with `alphagov/govuk-content-schemas` as the repo name, rather than `alphagov/collections`.

So, we should probably make jenkins.sh read the "REPO_NAME" environment variable, use `alphagov/collections` as a default, and make jenkins-schema.sh set `REPO_NAME` to `alphagov/govuk-content-schemas`